components/connection/WellKnownURLs.tsx

```diff
@@ -8,61 +8,86 @@ const links = [
       'Our OpenID configuration URI which your customers will need if they are connecting via OAuth 2.0 or Open ID Connect.',
     href: '/.well-known/openid-configuration',
+    buttonText: 'View',
   },
   {
     title: 'IdP Metadata',
     description:
       'The metadata file that your customers who use our SAML federation feature will need to set up SAML SP configuration on their application.',
     href: '/.well-known/idp-metadata',
+    buttonText: 'View',
   },
   {
     title: 'IdP Configuration',
     description:
       'The configuration setup guide that your customers who use our SAML federation feature will need to set up SAML SP configuration on their application.',
     href: '/.well-known/idp-configuration',
+    buttonText: 'View',
+  },
+  {
+    title: 'SAML Public Certificate',
+    description: 'The SAML Public Certificate if you want to enable encryption with your Identity Provider.',
+    href: '/.well-known/saml.cer',
+    buttonText: 'Download',
   },
 ];
 
-const WellKnownURLs = ({ className }: { className?: string }) => {
+type LinkCardProps = typeof links[number];
+
+const WellKnownURLs = () => {
   const { t } = useTranslation('common');
+
+  return (
+    <>
+      <div className='mb-5 flex items-center justify-between'>
+        <h2 className='font-bold text-gray-700 dark:text-white md:text-xl'>
+          {t('here_are_the_set_of_uris_you_would_need_access_to')}:
+        </h2>
+      </div>
+      <div className='space-y-3'>
+        {links.map((link) => (
+          <LinkCard
+            key={link.href}
+            title={link.title}
+            description={link.description}
+            href={link.href}
+            buttonText={link.buttonText}
+          />
+        ))}
+      </div>
+    </>
+  );
+};
+
+const LinkCard = ({ title, description, href, buttonText }: LinkCardProps) => {
   return (
-    <div className={className}>
-      <p>{t('here_are_the_set_of_uris_you_would_need_access_to')}:</p>
-      <br />
-      <ul className='flex flex-col space-y-1'>
-        {links.map((link) => {
-          return (
-            <li key={link.href} className='text-sm'>
-              <p>{link.description}</p>
-              <Link
-                href={link.href}
-                target='_blank'
-                rel='noreferrer'
-                className='link flex hover:link-primary'>
-                <ArrowRightOnRectangleIcon className='mr-1 h-5 w-5' aria-hidden /> {link.title}
-              </Link>
-              <br />
-            </li>
-          );
-        })}
-      </ul>
+    <div className='space-y-2 rounded-md border p-4 hover:border-gray-400'>
+      <div className='flex items-center justify-between'>
+        <div className='space-y-2'>
+          <h3 className='font-bold'>{title}</h3>
+          <p className='text-[15px]'>{description}</p>
+        </div>
+        <div className='mx-4'>
+          <Link href={href} target='_blank' rel='noreferrer' className='btn-outline btn-xs btn w-32'>
```

Comment:
Applied button style. Need to refactor the the existing component `LinkOutline` to accept `target`, `rel` as props.

Reply:
done.

Reply:
Thanks.